dev.alextren.Spot.desktop

```diff
@@ -0,0 +1,9 @@
+[Desktop Entry]
```

Comment:
Looks like this file is not being used now in the manifest, can you remove it?

Reply:
done!